easybuild/easyconfigs/n/netCDF/netCDF-4.4.1.1-intel-2017a-HDF5-1.10.1.eb

```diff
@@ -0,0 +1,35 @@
+name = 'netCDF'
+version = '4.4.1.1'
+hdf5_ver = '1.10.1'
+versionsuffix = '-HDF5-%s' % hdf5_ver
+
+homepage = 'http://www.unidata.ucar.edu/software/netcdf/'
+description = """NetCDF (network Common Data Form) is a set of software libraries 
+ and machine-independent data formats that support the creation, access, and sharing of array-oriented 
+ scientific data."""
+
+toolchain = {'name': 'intel', 'version': '2017a'}
+toolchainopts = {'pic': True, 'usempi': True}
+
+source_urls = ['https://github.com/Unidata/netcdf-c/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['7f040a0542ed3f6d27f3002b074e509614e18d6c515b2005d1537fec01b24909']
+
+dependencies = [
+    ('HDF5', hdf5_ver),
+    ('cURL', '7.54.0'),
+    ('Szip', '2.1'),
+]
+
+builddependencies = [
+    ('CMake', '3.8.2'),
+    ('Doxygen', '1.8.13'),
+]
+
+# make sure both static and shared libs are built
+configopts = [
+    "-DBUILD_SHARED_LIBS=OFF ",
+    "-DBUILD_SHARED_LIBS=ON ",
```

Comment:
Yes, it basically makes EasyBuild perform the configure-build-install cycle twice: the first time with `-DBUILD_SHARED_LIBS=OFF` during configure, the 2nd time with `-DBUILD_SHARED_LIBS=ON`.

That seems to be the only way to get both shared and static libraries in the same installation...